collectors/python.d.plugin/nginx/nginx.chart.py

```diff
@@ -61,12 +61,12 @@ def _get_data(self):
         """
         try:
             raw = self._get_raw_data().split(" ")
-            return {'active': int(raw[2]),
-                    'requests': int(raw[9]),
-                    'reading': int(raw[11]),
-                    'writing': int(raw[13]),
-                    'waiting': int(raw[15]),
-                    'accepts': int(raw[7]),
-                    'handled': int(raw[8])}
+           return {'active': int(raw[2]),
+                    'requests': int(raw[10]),
+                    'reading': int(raw[12]),
+                    'writing': int(raw[14]),
+                    'waiting': int(raw[16]),
+                    'accepts': int(raw[8]),
+                    'handled': int(raw[9])}
```

Comment:
i think we need an `if` there. if `request_time` in raw or based on len(raw). To not break parsing old/not tengine stub status.

```python
if condition:
    return old
return new
```

Reply:
modify 'requests':str(raw[11]),
ValueError: invalid literal for int() with base 10: '155535228875\nReading:


mybe,status respon 
Active connections: 240 \nserver accepts handled requests request_time\n 21329772 21329772 110166559 155336225319\nReading: 0 Writing: 166 Waiting: 78

Reply:
raw = self._get_raw_data().replace("\n"," ").split(" ")
            return {'active': int(raw[2]),
                    'requests':int(raw[13]),
                    'reading': int(raw[15]),
                    'writing': int(raw[17]),
                    'waiting': int(raw[19]),
                    'accepts': int(raw[11]),
                    'handled': int(raw[12])}